README.md

```diff
@@ -76,6 +78,50 @@ pipeline:
     command: echo "hello, world, command_stage_3"
 ```
 
+HCL (all files with .hcl prefix are assumed to be of HCL format)
+```
+//The pipeline
+pipeline {
+  stage {
+    name = "command_stage_1"
+    type = "command"
+    command = "echo 'hello, world'"
+  }
+  stage {
+    name = "command_stage_2"
+    type = "command"
+    command = "echo 'hello, world, command_stage_2'"
+  }
+  stage {
+    name = "command_stage_3"
+    type = "command"
+    command = "echo 'hello, world, command_stage_3'"
+  }
+}
+```
+
+JSON
+```json
+{
+  "pipeline":{
+    "stage" :{
+      "name":"command_stage_1",
+      "type":"command",
+      "command":"echo 'hello, world'"
+    },
+    "stage":{
```

Comment:
Good point, interestingly the HCL lib accepts it and parses correctly, but in real terms javascript would parse this and assume the state of the last key. Maybe we could change the top level object to ```stages``` and have an array under that. I can change it, but what are your thoughts?